protocol/signalfx/format/log/signalfx_log.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+package sfx_log_format;
+
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "gen_go/;log_format";
+
+// model is based off https://github.com/open-telemetry/oteps/blob/master/text/0097-log-data-model.md#log-and-event-record-definition
+message LogRecord {
+    google.protobuf.Timestamp Timestamp = 1;
+    string TraceID = 2;
+    string SpanID = 3;
+    string TraceFlags = 4;
+    string SeverityTest = 5;
+    string Name = 6;
+    AttributeValue Body = 7;
+    map<string, AttributeValue> Resource = 8;
+    map<string, AttributeValue> Attributes = 9;
+}
+
+message AttributeValue {
+    oneof value {
```

Comment:
`oneof` is pretty slow. We opted not to use it in OTLP because of that.

Reply:
Thanks i will add a TODO... to benchmark the proto

Reply:
i moved it to its own message as we want to know the ValueType which will be used for counting the no.of messages